lib/actions/addClass.js

```diff
@@ -0,0 +1,30 @@
+class Add {
```

Comment:
I think we should use a more descriptive name for this class. As an outsider, I'm left wondering "what is this adding?" AddLink? Also, the filename should follow suite and doesn't need the word `Class` in the filename.

Reply:
Got emmm! Great point. 👍